app/src/pages/inside/logsPage/logsGridToolbar/logsGridToolbar.jsx

```diff
@@ -258,7 +255,7 @@ export class LogsGridToolbar extends Component {
               >
                 {Parser(MarkdownIcon)}
               </button>
-              {isLogView && (
+              {!isNestedStepsView && (
```

Comment:
Maybe better to create a constant and name it isLogView?
As for me, it is clear in what view we need to display this or that layout.
What do you think?

Reply:
The var `isLogView` was wrong in that meaning, because actually it was representing the `viewWithoutNestedSteps`.
On the other hand `isLogView` name is not relevant, because the `logsGridToolbar` always on the log view :)

Reply:
And according to the requirements, we need to hide some controls when there are no nested steps, as these controls valid only for them.

Reply:
OMG!)
Okay)